src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -63,7 +63,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String password = request.getParameter("password");
     String hashedPassword = BCrypt.hashpw(password, BCrypt.gensalt());
-    String aboutMe = request.getParameter("aboutMe");
```

Comment:
Do we only want to default to an empty string if the request param is null? Or are we intentionally always setting about me as an empty string?